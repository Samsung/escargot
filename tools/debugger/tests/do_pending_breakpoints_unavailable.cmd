b not_existing_func
y
b 22
y
n
n
b not_existing.js:6
y
n
