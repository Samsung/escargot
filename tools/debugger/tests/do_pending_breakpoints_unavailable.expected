Connecting to: localhost:6501
Connection created!!!
Stopped at tools/debugger/tests/do_pending_breakpoints_unavailable.js:20
(escargot-debugger) b not_existing_func
No breakpoint found, do you want to add a pending breakpoint? (y or [n])
Pending breakpoint 1 at not_existing_func()
(escargot-debugger) b 22
No breakpoint found, do you want to add a pending breakpoint? (y or [n])
Pending breakpoint 2 at 22()
(escargot-debugger) n
Stopped at tools/debugger/tests/do_pending_breakpoints_unavailable.js:21
(escargot-debugger) n
Stopped at tools/debugger/tests/do_pending_breakpoints_unavailable.js:22
(escargot-debugger) b not_existing.js:6
No breakpoint found, do you want to add a pending breakpoint? (y or [n])
Pending breakpoint 3 at not_existing.js:6
(escargot-debugger) n
Connection closed.
